src/org/zaproxy/zap/extension/ascanrulesAlpha/ZapAddOn.xml

```diff
@@ -8,6 +8,7 @@
 	<![CDATA[
 	Updated add-on's info URL.<br>
 	Minor code changes.
```

Comment:
This line should be terminated with "<br>", for proper formatting when shown in ZAP.